### PR TITLE
Add display id to disruptions to uniquely identify disruptions

### PR DESCRIPTION
**Pull request by alvak1**
Testing instructions

- Create a new disruption or edit an existing disruption should create a display id. This is visible when you export data in view-all-disruptions page or visible in dashboards page

### CODE REVIEW

File: site/utils/siri.ts [hunk 1]
```diff
@@ -127,7 +127,16 @@ export const getPtSituationElementFromDraft = (disruption: Disruption) => {
                           Severity: consequence.disruptionSeverity,
                           Affects: {
                               ...(consequence.consequenceType === "networkWide" ||
-                              consequence.consequenceType === "operatorWide"
+                              consequence.consequenceType === "stops"
+                                  ? {
+                                        Operators: {
+                                            AllOperators: "",
+                                        },
+                                    }
```
Comment (alainajaffer) [inline site/utils/siri.ts:135]: why is this change here?

**Comment by alainajaffer** [comment]
None of my old disruptions are showing

**Comment by alvak1** [comment]
> None of my old disruptions are showing

Yes thats cause displayId is a mandatory field and disruptions without this field will fail parsing

File: site/utils/siri.ts (hunk 1 above)
Comment (alvak1) [reply site/utils/siri.ts]: Fixed after merging with main
